partitions.md

```diff
@@ -17,11 +17,15 @@ Without specifying a partition, I've been able to allocate 30 GB but not
 
 We have access to:
 
-* bml - low priority big mem (up to a TB)
-* bmm - medium priority big mem
-* bmh - high priority big mem
+* bml - low priority big mem node (up to a TB)
+* bmm - medium priority big mem node
+* bmh - high priority big mem node
 
-as well as some default queue/partition that I don't yet understand :)
+as well as
+
+* low2 - low priority compute nodes
```

Comment:
add "(up to 256Gb)"

Reply:
I see you specified below, so I suggest adding or deleting the (up to a TB) on line 20